src/main/java/uk/gov/ons/census/casesvc/service/CaseService.java

```diff
@@ -60,6 +70,35 @@ public Case saveCase(CreateCaseSample createCaseSample) {
     return caze;
   }
 
+  public Case buildCCSCase(
+      String caseId, SampleUnitDTO sampleUnit, String actionPlanId, String collectionExerciseId) {
```

Comment:
Move the action plan and collex IDs into this class and they won't have to be passed in as parameters

Reply:
done